Food Ordering System/style.css

```diff
@@ -0,0 +1,329 @@
+*{
+    font-family: 'Poppins', sans-serif;
+}
+
+body{
+    margin:0 auto 100%;
+    padding: 0;
+    overflow-x:hidden;
+    width: 100%;
+    height: 100%;
+}
+/* Header */
+#cart-logo{
+    padding: 20px 30px;
+    margin-left:20px; 
+    width: 40px;
+    background-image: url("img/cartlogo.png") ;
+    background-size: auto 100%;
+    border-radius:10%;
+    /* background-color: rgba(255, 255, 255, 0.5); */
+    background-size: cover;
+}
+
+#cart-button:hover{
+    background-color: #edc531;
+}
+
+.header{ 
+    height: 450px;
+    background-image: url('img/header.jpg');
+    background-size: auto 100%;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover;
+    } 
+
+.logo img{
+    margin-top: 10px;
+    margin-left:30px;
+    height: 100px;
+    width: 100px;
+    max-width: 100%;
+    max-height: 100%;
+    object-fit: cover;
+}
+/* Logo Text */
+#eureka {
+    color:white;
+    margin-top:0;
+    margin-left:20px;
+    font-style: italic;
+    font-family: 'Dancing Script', cursive;
+    font-size: 25px;
+    opacity:0.6;
+}   
+
+.logo-nav{
+    height: 70px;
+    margin-left: 50px;
+    margin-right: 60px;
+    display:flex;
+    justify-content: space-between;
+    padding: 10px 10px;   
+}
+
+.navigation ul li{
+    display:inline-block ;
+    padding-top: 0.8em;
+    padding-left: 1em;
+   font-weight: bold;
+   color: white;
+}
+
+ul li:hover{
+    color:#c9cba3;
+    transition: 0.35s ease-in-out;
+    cursor: pointer;
+}
+
+h1{
```

Comment:
don't override the native HTML component.

Reply:
Not even main and footer?

Reply:
You can override main because that is used in only one place. Even though I didn't see a page with 2 footers elements, you can have 2 footers 😂